cqrs-user-command-service/src/main/java/dev/amir/usercommand/UserCommandServiceApplication.java

```diff
@@ -1,12 +1,15 @@
 package dev.amir.usercommand;
 
+import lombok.extern.slf4j.Slf4j;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
+@Slf4j
 @SpringBootApplication
 public class UserCommandServiceApplication {
 
     public static void main(String[] args) {
+        log.info("Starting User Command Service application.");
```

Comment:
this log is not part of the application layer (application folder in the UserCommandService project)

Reply:
"The required changes have been resolved."